tests/testAutopas/tests/selectors/tuningStrategy/PredictiveTuningTest.cpp

```diff
@@ -27,10 +27,10 @@ TEST_F(PredictiveTuningTest, testSearchSpaceOneOption) {
 }
 
 TEST_F(PredictiveTuningTest, testSearchSpaceMoreOptions) {
```

Comment:
Remove tests that are now in TuningStrategyTest.cpp
- testSearchSpaceEmpty
- testSearchSpaceOneOption
- testSearchSpaceMoreOptions
- testRemoveN3OptionRemoveAll
- testRemoveN3OptionRemoveSome